Projects/Claims/Sources/Models/FlowClaimSingleItemCheckoutStepModel.swift

```diff
@@ -50,4 +50,20 @@ struct ClaimFlowMoneyModel: Codable, Equatable {
         formatter.currencySymbol = self.currencyCode
         return formatter.string(for: amount) ?? ""
     }
+
+    func formatAndPrintDouble(value: Double) -> String {
+        let formatter = NumberFormatter()
+        formatter.groupingSeparator = " "
+        formatter.numberStyle = .decimal
+
+        if value.truncatingRemainder(dividingBy: 1) == 0 {
+            formatter.minimumFractionDigits = 0
+            formatter.maximumFractionDigits = 0
+        } else {
+            formatter.minimumFractionDigits = 1
+            formatter.maximumFractionDigits = 2
+            formatter.decimalSeparator = "."
```

Comment:
I don't think that we should define decimal seperator.
Also I don't think you need to check if  truncatingRemainder == 0. Just need to set minimumFractionDigits to 0 and maximum to 2